main.tf

```
@@ -1,3 +1,7 @@
+locals {
+  connect_vcs_repo = (var.repository_owner == null || var.repository_name == null) ? {} : { create = true }
```

Small tweak, but would prefer this was the other way around:

```suggestion
  connect_vcs_repo = (var.repository_owner != null && var.repository_name != null) ? { create = true } : {}
```

---

I actually think the original is easier to understand

---

It's just my preference to use `&&` and `!= null` over `||`, I wouldn't oppose if it was reverted. 😉

We'll let @svanharmelen be the decider...

---

I agree with you @shoekstra...